### ISSUE TITLE
Create shorthand methods for every HTTP method

### ISSUE THREAD

**Issue by amitguptagwl** [NaturalIntelligence/anumargak#1]
> This issue is raised for developers who are just starting with open source development. So please refrain yourself if you're experienced.
 
Currently a route can be registered using `on` method.

```js
const router = require('anumargak')();
router.on("GET","/test/url",calback);
```

Create shorthand methods to register a route in following ways

```js
const router = require('anumargak')(https://github.com/NaturalIntelligence/anumargak/tree/master/src/letsRoute.js);
router.get("/test/url",calback);
```

**HInt**
Add a method to file [letsRoute.js]()

```js
/**
 * Adds routes for GET method and URL
 * @param {string} url 
 * @param {function} fn 
 */
Anumargak.prototype.get = function(url,fn){
  this.on("GET",url,fn);
}
```

Add the methods for following httpMethods

* GET
* HEAD
* PUT
* POST
* DELETE

Don't forget to write unit tests. You may create separate test file as well. 

Feel free to leave a comment to discuss any doubt or question.

**Comment by rajeshdh** [comment]
Can I take this?

**Comment by amitguptagwl** [comment]
Yes please go on.

> [Bookmark](https://github.com/NaturalIntelligence/anumargak/stargazers) this repository for easy discovery
> Be the part of [team](https://github.com/socialat/team/issues/1) to support opensource community

**Comment by rajeshdh** [comment]
Should I update the tests with the new shorthand methods?

**Comment by amitguptagwl** [comment]
You should add more tests to test new shorthand methods.
